shubhali/curdop/routes/index.js

```diff
@@ -0,0 +1,61 @@
+var express = require('express');
+var router = express.Router();
+var empModel=require('../modules/employee');
+var employee=empModel.find({});
+
+/* GET home page. */
+router.get('/', function(req, res,next) {
+  employee.exec(function(err,data){
+    if(err) throw err;
+    res.render('index', { title: 'Employee Records',records:data });
+  })
+});
+router.post('/', function(req, res, next) {
+  var empDetails = new empModel({
+    name: req.body.uname,
+    email: req.body.email,
+    lastname: req.body.lastname,
+    phone: req.body.phone,
+    field: req.body.field,
+    city: req.body.city,
+  });
+ 
+  empDetails.save(function(err,req1){
+    if(err) throw err;
+    employee.exec(function(err,data){
+      if(err) throw err;
+      res.render('index', { title: 'Employee Records', records:data, success:'Record Inserted Successfully' });
+        });
+  })
+});
+router.get('/delete/:id',function(req,res,next){
+  var id=req.params.id;
+  var del=empModel.findByIdAndDelete(id);
+  del.exec(function(err){
+    if(err) throw err;
+    res.redirect('/');
+  });
+});
+router.get('/edit/:id',function(req,res,next){
+  var id=req.params.id;
```

Comment:
follow coding rules like space after comma, space before and after operator, space after colon